src/utils/test/helpers.unit.test.ts

```
@@ -0,0 +1,54 @@
+import { SUPPORTED_COUNTRIES } from '../../config';
+import { PublicHoliday, PublicHolidayShort } from '../../types';
+import { validateInput, shortenPublicHoliday } from '../helpers';
+
+describe('=== HELPERS UNIT ===', () => {
+	describe('> validateInput', () => {
+		it('should return true for valid inputs', () => {
+			const country = SUPPORTED_COUNTRIES[0];
+			const year = new Date().getFullYear();
+
+			expect(validateInput({ year, country })).toBeTruthy();
+		});
+
+		it('should throw an error for unsupported country', () => {
+			const unsupportedCountry = 'KG';
+
+			expect(() => validateInput({ country: unsupportedCountry })).toThrow(
+				new Error(`Country provided is not supported, received: ${unsupportedCountry}`),
+			);
+		});
+
+		it('should throw an error for invalid year', () => {
+			const year = 2000;
+
+			expect(() => validateInput({ year })).toThrow(
+				new Error(`Year provided not the current, received: ${year}`),
+			);
+		});
```

The function is throwing error if the year is not a current year. So we should test that as well. I would prefer to create two test cases:
- `should throw an error for a year that is in the feature`. And I would use ` const nextYear = new Date().getFullYear() + 1;` for that. 
- `should throw an error for a year that is in the past`. And I would use ` const nextYear = new Date().getFullYear() - 1;` for that.